### DISCUSSION TITLE
Are hotkeys on macOS intended to use Control?

### DISCUSSION THREAD

**Discussion by OfficialCRUGG** [ppy/osu#25554] [Q&A]
When I hover over e.g. the settings, it shows up "Ctrl+O" has the hotkey. This is also the keys I need to press. I assume the hotkey is the same on Windows, where it makes sense. But I wonder why it's also like this on macOS - since here ⌘ (Command) is usually used in place of Window's Control key - even if macOS still has a Control key.
Is there any particular reason this is the case?

**Comment by peppy** [comment]
You can re-bind these if you'd like. For now we're sticking with ctrl for all game keys (except things like copy/paste etc.).
